splink/comparison_level_library.py

```diff
@@ -68,3 +70,28 @@ def create_sql(self, sql_dialect: SplinkDialect) -> str:
 
     def create_label_for_charts(self) -> str:
         return f"Levenshtein distance of {self.col_name} <= {self.distance_threshold}"
+
+
+class JaroWinklerLevel(ComparisonLevelCreator):
+    def __init__(self, col_name: str, distance_threshold: Union[int, float]):
+        """A comparison level using a Jaro-Winkler distance function
+
+        e.g. `jaro_winkler(val_l, val_r) <= distance_threshold`
+
+        Args:
+            col_name (str): Input column name
+            distance_threshold (Union[int, float]): The threshold to use to assess
+                similarity
+        """
+        self.col_name = col_name
```

Comment:
maybe we could do this via `super().__init__(col_name)` as will make it easier if we want to rename / change the structure of this

Reply:
Yeah, for now, I think it makes sense to have them all using the super class.

See my comment here on this - https://github.com/moj-analytical-services/splink/pull/1705#discussion_r1386694378.